assets/dev/js/editor/views/preview.js

```diff
@@ -86,11 +94,11 @@ const Preview = BaseSectionsContainerView.extend( {
 		$contentContainer.html( this.$childViewContainer );
 
 		if ( elementor.userCan( 'design' ) ) {
-			const addNewSectionView = new AddSectionView();
+			this.addNewSectionView = new AddSectionView();
 
-			addNewSectionView.render();
+			this.addNewSectionView.render();
 
-			$contentContainer.append( addNewSectionView.$el );
+			$contentContainer.append( this.addNewSectionView.$el );
```

Comment:
Why?

Reply:
Will revert this change, not needed anymore.